docs/.circleci/config.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
```

Comment:
This file isn't really needed, mkdocs doesn't include arbitrary files in the final build anyway

Reply:
Thought so, I had listed as a TODO to remove it but kept it around just in case. Will remove it.